omnibus-nikos_arm64/Dockerfile

```diff
@@ -13,23 +13,36 @@ FROM arm64v8/ubuntu:16.04
 
 # Build Args
 ARG GIMME_GO_VERSION=1.17.6
+ARG CMAKE_VERSION=3.23.0
+ARG CMAKE_SHA256="9f8d42ef0b33d1bea47afe15875435dac58503d6a3b58842b473fd811e6df172"
 ARG CLANG_VERSION=8.0.0
 ARG CLANG_SHA256="a77eb8fde0a475c25d46dccdeb851a83cbeeeb11779fa2218ae19db9cd0e51f9"
 
 # Environment
 ENV GOPATH /go
 ENV GIMME_GO_VERSION $GIMME_GO_VERSION
+ENV CMAKE_VERSION $CMAKE_VERSION
+ENV CMAKE_SHA256 $CMAKE_SHA256
 ENV CLANG_VERSION $CLANG_VERSION
 ENV CLANG_SHA256 $CLANG_SHA256
 
-RUN apt-get update && apt-get install -y fakeroot cmake curl bzip2 g++ gcc git \
+RUN apt-get update && apt-get install -y fakeroot procps bzip2 g++ gcc git \
```

Comment:
`curl` is used below, so should still be required, yes?

Reply:
The first lines of the Dockerfile already install a specific, pinned version of curl, so we don't depend on the system one. This one shouldn't be needed.

Reply:
👍